example/ios/Podfile.lock

```diff
@@ -468,6 +477,17 @@ EXTERNAL SOURCES:
   Yoga:
     :path: "../node_modules/react-native/ReactCommon/yoga"
 
+CHECKOUT OPTIONS:
```

Comment:
just curious, why do we need this if we also define branches?

Reply:
this is auto-generated, it determines the exact versions that cocoapods resolved to. This way, every developer is guaranteed to be using the exact same version (in this case, a particular commit), even if the dependency is just defined as a branch.

Reply:
this is useful because if a given commit could break the app for example